### ISSUE TITLE
Unable to preview project on mobile device

### ISSUE THREAD

**Issue by RoKrypto** [ritwickdey/vscode-live-server#149]
## I'm submitting a...

```html
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  <!-- Please search GitHub for a similar issue or PR before submitting -->
[ ] Feature request
[ ] Documentation issue or request
[x] Other: Feature malfunction
```

## Current behavior

Live Server works just fine to preview my project on the browser from my computer. The issue I have is that I'm unable to preview it from my mobile device (Samsung Galaxy S5, my browser is Google Chrome). I followed the instructions from the FAQ section but I didn't succeed. The preview takes so much to load. Finally, it ends up showing this message:

You can't access this site.
192.168.xx.xx took so much to reply.
Look for 192.168 5500 index in Google.
ERR_CONNECTION_TIMED_OUT

## Expected behavior

Being able to preview my project on the browser from my phone.

## Environment

```html
Browser:

- Chrome (mobile) version 67.0.3396.81

For Tooling issues:

- Live Server: 4.1.0
- Desktop Platform: Windows 10
- Mobile Platform:  Android 6.0.1; SM-G900H Build/MMB29K
- Visual Studio Code: 1.24.0
```

**Comment by rjoydip** [comment]
Enable `"liveServer.settings.useLocalIp": true` and try again.

**Comment by ritwickdey** [comment]
Sometimes, the firewall blocks incoming requests. Try to turn off firewall temporarily & try again.

**Comment by RoKrypto** [comment]
It worked! 💃 I kinda dislike turning off the firewall though, but it seems it's the only way to make it work. Thank you, Ritwick.

**Comment by ritwickdey** [comment]
Actually, you don't have to turn off the firewall. I never did, but it's working on my laptop. 

I think you have to turn off/on any rules of the firewall. Just check on the internet.

**Comment by RoKrypto** [comment]
I'll research about that. Thanks once again! 😃

**Comment by Caiqueapenas** [comment]
[06/16/2020]
Thank so much. it's perfectlly woriking. :)

**Comment by ms-studio** [comment]
In my case, I encountered this issue on MacOS.

I activated `"liveServer.settings.useLocalIp": true`. This gave me an IP looking like http://192.168.95.1:5500/ (instead of 127.0.0.1:5500).

Still I got an error message when connecting over mobile.

Then I went into the MacOS Preferences > Network. There it showed "Wiki is connected to *NetworkName* with IP address 192.168.0.227.

So I entered http://192.168.0.227:5500/ on my mobile device and BAM!, it worked :)
